svelte.config.js

```diff
@@ -13,51 +12,16 @@ const config = {
 	kit: {
 		adapter: adapter(),
 
-		// hydrate the <div id="svelte"> element in src/app.html
-		target: '#svelte',
+		prerender: {
+			default: true,
+		},
 
 		paths: {
 			base: isDev ? undefined : '/commercial-templates',
 		},
 
 		// Github pages really likes its trailing slashes!
 		trailingSlash: isDev ? 'never' : 'always',
-
-		vite: defineConfig({
-			plugins: [
-				{
-					name: 'watch-templates',
-					configureServer(server) {
-						server.watcher.add('/src/templates');
```

Comment:
Do we not need to pass `/src/templates` for the vite watcher anymore?

Reply:
Haha omg you're absolutely right! Thanks for the spot 👍 

It's 2022 and we still have to write our own watch plugins? 😮